plugins/qCompass/ccCompass.cpp

```diff
@@ -0,0 +1,964 @@
+//##########################################################################
+//#                                                                        #
+//#                       CLOUDCOMPARE PLUGIN: ccCompass                   #
+//#                                                                        #
+//#  This program is free software; you can redistribute it and/or modify  #
+//#  it under the terms of the GNU General Public License as published by  #
+//#  the Free Software Foundation; version 2 of the License.               #
+//#                                                                        #
+//#  This program is distributed in the hope that it will be useful,       #
+//#  but WITHOUT ANY WARRANTY; without even the implied warranty of        #
+//#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         #
+//#  GNU General Public License for more details.                          #
+//#                                                                        #
+//#                             COPYRIGHT: XXX                             #
+//#                                                                        #
+//##########################################################################
+
+#include "ccCompass.h"
+
+
+//Qt
+#include <QtGui>
+
+ccCompass::ccCompass(QObject* parent/*=0*/)
```

Comment:
I mean, this happens if you never call the qCompass plugin during the cc execution. the pointer remains not initialized and if(ptr) will return true...